calcom/cal.com
# fix: added aria-label to timezone input

## What does this PR do?
This addresses an accessibility issue with the timezone input element by adding an aria-label for screen readers to announce that input upon focus. 



- https://github.com/calcom/cal.com/issues/17186



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?


1. Run Google Lighthouse accessibility check to verify that the input element no longer triggers warnings for missing Input element labels.
2. Use the browser's developer tools to inspect the timezone input and confirm that the aria-label="Timezone Selection" is applied correctly.

- Are there environment variables that should be set? -NA
- What are the minimal test data to have? 
   Go to event Booking page and run lighthouse test for accessibility.
- What is expected (happy path) to have (input and output)?
   The timezone selection input should be accessible to screen readers, and no errors should be flagged by accessibility audits.
